core/src/test/java/forklift/consumer/MessageRunnableTest.java

```diff
@@ -44,33 +70,37 @@ else if (m.isAnnotationPresent(OnValidate.class))
 
         TestConsumer1 tc = new TestConsumer1();
 
-        MessageRunnable mr = new MessageRunnable(null, new ForkliftMessage(jmsMsg), null, tc, onMessage, onValidate, null, createOnProcessStepMap(), Collections.emptyList());
+        MessageRunnable
```

Comment:
Not really sure of the point of putting the variable type and identifier on different lines when there is enough room?